sql_joins/sql_joins.md

```diff
@@ -0,0 +1,875 @@
+}
+
+.circle {
+  border: .2em solid #000;
+  border-radius: 50%;
+  width: 24em;
+  height: 24em;
+  text-align: center;
+  vertical-align: center;
+  position: absolute;
+}
+
+.circle p {
+  height: 24em;
+  width: 14em;
+  display: table-cell;
+  vertical-align: middle;
+}
+
+#c1 {
+  top: 0;
+  left: 0;
+  background-color: rgba(255,0,0,.25);
+}
+
+#c2 {
+  top: 0;
+  left: 12em;
+  background-color: rgba(0,255,0,.25);
+}
+
+#c1 p{
+  padding-right: 2em;
+}
+#c2 p {
+    padding-left: 12em;
+    width: 22em;
+}
+
+
+#j12 {
+    position: absolute;
+    left: 12em;
+    width: 11em;
+    height: 22em;
+    padding: 1em;
+    text-align: center;
+}
+
+#j12 p {
+  height: 22em;
+  width: 10em;
+  display: table-cell;
+  vertical-align: middle;
+}
+</style>
+<div id="infographic">
+  <div id="c1" class="circle">
+    <p>Left Side: <br/>Only Math Grades</p>
+  </div>
+
+  <div id="c2" class="circle">
+    <p>Right Side: <br/>Only Language Grades</p>
+  </div>
+
+  <div id="j12" class="joined">
+    <p>
+      Overlap: <br/>Both Math and Language Grades
+    </p>
+  </div>
+</div>
+
+</lia-keep>
+
+There are 4 basic join types that can be used.  We'll go into more detail about how each one works in the next few pages.  
+
+Here, we simply provide a visual highlight to indicate the part of the data that's considered for inclusion with each type of join.
+
+* **`INNER JOIN`**.  ![Overlapping circles with only the inner, overlapping part highlighted](media/inner.png) 
+
+
+* **`LEFT JOIN`**.  ![Overlapping circles with the left circle, including the overlapping part of the left circle, highlighted](media/left_outer.png) 
+
+
+* **`RIGHT JOIN`**.  ![Overlapping circles with the right circle, including the overlapping part of the right circle, highlighted](media/right_outer.png) 
+
+* **`FULL JOIN`**.  ![Overlapping circles with both circles highlighted](media/full_outer.png) 
+
+
+#### `INNER JOIN`
+
+Here, let's consider the left table to be **math\_grades** and the right table to be **language\_grades**.
+
+![Overlapping circles with only the inner, overlapping part highlighted](media/inner.png) 
+
+**What is it?**
+-----
+
+An `INNER JOIN` (and this is the default behavior of `JOIN` without any modifying words) finds matching grade data falling in the "Both Math and Language Grades" overlap section.  This is often the data you want to capture.  
+
+**Why use it?**
+-----
+
+If you're conducting research on the correlation between math and language grades, this is the join you want.  If a student lacks one or the other grade, their data isn't useful to you, so you don't want it. 
+
+**Results**
+-----
+The result of an inner join will be a table that only has rows of data for students who appear in both tables.  If a student is missing in one or the other table of grades, that student won't appear in your result set.
+
+**Code**
+----
+
+An `INNER JOIN` shows up in code like this:
+
+```sql
+...
+FROM math_grades INNER JOIN language_grades
```

Comment:
```suggestion
FROM math_grades INNER JOIN language_grades
...
```
Is it possible to join tables without an ON or a USING statement? If so, the second ellipses might not be necessary, but I think there are.